.github/workflows/ci-cd.yml

```diff
@@ -451,7 +451,7 @@ jobs:
 
   CI-status-report:
     runs-on: ubuntu-latest
-    needs: [rspec, jest, cypress, build_test]
+    needs: [rspec, jest, cypress, build_test, storybook, audit]
```

Comment:
I merged a broken build by mistake, and this made me realize I forgot to have storybook and audit as part of the CI-status checks